src/StoryTime.cpp

```diff
@@ -100,6 +104,9 @@ struct Game
 
     // Show window
     window.display();
+    
+    currentSlice->drawComplete();
```

Comment:
currentSlice draw should just call this itself at end of draw method. Then drawComplete can be private